src/View/Helper/FormHelper.php

```diff
@@ -260,30 +260,30 @@ public function widgetRegistry(WidgetRegistry $instance = null, $widgets = [])
      */
     protected function _addDefaultContextProviders()
     {
-        $this->addContextProvider('orm', function ($request, $data) {
+        $this->addContextProvider('orm', function ($request, $data, $requestType) {
```

Comment:
Sorry, those are actually artefacts left from a different approach to this problem I tried before. I will remove those completely...